examples/antd-calendar.js

```diff
@@ -174,6 +174,7 @@ class Demo extends React.Component {
                     tabIndex="-1"
                     className="ant-calendar-picker-input ant-input"
                     value={value && value.format(getFormat(state.showTime)) || ''}
+                    inputMode={"numeric"}
```

Comment:
here ⤴️

Reply:
done. I also remove it from other examples as well
Sorry about that